WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Grouped Products/GroupedProductListSelectorDataSource.swift

```diff
@@ -0,0 +1,112 @@
+        return lhsProductIDIndex < rhsProductIDIndex
+    }
+
+    // Observable list of the latest grouped product IDs
+    var productIDs: Observable<[Int64]> {
+        productIDsSubject
+    }
+    private let productIDsSubject: PublishSubject<[Int64]> = PublishSubject<[Int64]>()
+
+    private(set) var groupedProductIDs: [Int64] = [] {
+        didSet {
+            if groupedProductIDs != oldValue {
+                productIDsSubject.send(groupedProductIDs)
+            }
+        }
+    }
+
+    var selected: Product?
+
+    private let siteID: Int64
+    private let product: Product
+    private let imageService: ImageService
+
+    init(product: Product, imageService: ImageService = ServiceLocator.imageService) {
+        self.siteID = product.siteID
+        self.product = product
+        self.groupedProductIDs = product.groupedProducts
+        self.imageService = imageService
+    }
+
+    func createResultsController() -> ResultsController<StorageProduct> {
+        let storageManager = ServiceLocator.storageManager
+        let predicate = NSPredicate(format: "siteID == %lld AND productID IN %@", siteID, groupedProductIDs)
+        return ResultsController<StorageProduct>(storageManager: storageManager,
+                                                 matching: predicate,
+                                                 sortOrder: .nameAscending)
+    }
+
+    func handleSelectedChange(selected: Product) {
+        // no-op
+    }
+
+    func isSelected(model: Product) -> Bool {
+        return model == selected
+    }
+
+    func configureCell(cell: ProductsTabProductTableViewCell, model: Product) {
+        cell.selectionStyle = .none
+
+        let viewModel = ProductsTabProductViewModel(product: model)
+        cell.update(viewModel: viewModel, imageService: imageService)
+
+        let deleteButton = UIButton(type: .detailDisclosure)
+        deleteButton.setImage(.deleteCellImage, for: .normal)
+        deleteButton.tintColor = .systemColor(.tertiaryLabel)
+        deleteButton.on(.touchUpInside) { [weak self] _ in
+            self?.deleteProduct(model)
+        }
+        cell.accessoryView = deleteButton
+    }
+
+    func sync(pageNumber: Int, pageSize: Int, onCompletion: ((Bool) -> Void)?) {
+        let action = ProductAction.retrieveProducts(siteID: siteID,
+                                                    productIDs: groupedProductIDs,
+                                                    pageNumber: pageNumber,
+                                                    pageSize: pageSize) { result in
+                                                        switch result {
+                                                        case .success:
+                                                            onCompletion?(true)
+                                                        case .failure(let error):
+                                                            DDLogError("⛔️ Error synchronizing products: \(error)")
```

Comment:
```suggestion
                                                            DDLogError("⛔️ Error synchronizing grouped products: \(error)")
```

Reply:
yup! updated in https://github.com/woocommerce/woocommerce-ios/pull/2505/commits/60177e57082e3947d5a4da86f323dbb56e11e810